Makefile

```diff
@@ -10,31 +13,44 @@ compile:
 image: compile
 	operator-sdk build $(IMAGE_TAG)
 
+.PHONY: test
+test:
+	@echo $(CRDS)
+
+.PHONY: bundle
+bundle: image $(CRDS)
+	@echo "Bundle prepared, use operator-courier to push it to Quay"
+
+%.crd.yaml:
+	cp -f $(CRD_DIR)/$@ bundle/
+
 .PHONY: clean
-clean:
+clean: clean-bundle
 	rm -rf build/_output
 
+.PHONY: clean-bundle
+clean-bundle:
+	rm -f bundle/$(CRDS)
+
+
+
+
+#########################################
+# "Local" (ex. MiniShift) testing targets #
+#########################################
+
 .PHONY: deploy
 deploy: undeploy
 	oc create -f deploy/operator_service_account.yaml
-	oc create -f deploy/exposecontroller_service_account.yaml
 	oc create -f deploy/operator_role.yaml
-	oc create -f deploy/exposecontroller_role.yaml
 	oc create -f deploy/operator_role_binding.yaml
-	oc create -f deploy/exposecontroller_role_binding.yaml
-	oc create -f deploy/exposecontroller_role_binding_cluster_admin.yaml
-	oc create -f deploy/exposecontroller_role_binding_cluster_reader.yaml
-	oc create -f deploy/crds/rhjmc_v1alpha1_flightrecorder_crd.yaml
-	oc create -f deploy/crds/rhjmc_v1alpha1_containerjfr_crd.yaml
+	oc create -f deploy/crds/flightrecorders.rhjmc.redhat.com.crd.yaml
+	oc create -f deploy/crds/containerjfrs.rhjmc.redhat.com.crd.yaml
```

Comment:
After the rename of the CRD files, `operator-sdk generate openapi` does not seem to update them in response to changes in *_types.go. It might be best just copy each CRD manually, and leave them named as whatever the tooling expects.